token/program-2022/src/extension/confidential_transfer_fee/mod.rs

```diff
@@ -34,6 +40,9 @@ pub struct ConfidentialTransferFeeConfig {
     /// fee parameters, the withheld fee amounts can reveal information about transfer amounts.
     pub withdraw_withheld_authority_elgamal_pubkey: ElGamalPubkey,
 
+    /// If `false`, the harvest of withheld tokens to mint is rejected.
+    pub harvest_to_mint_disabled: PodBool,
```

Comment:
nit: Let's flip it for ease of understanding, because the comment seems to be incorrect
```suggestion
    /// If `false`, the harvest of withheld tokens to mint is rejected.
    pub harvest_to_mint_enabled: PodBool,
```

Reply:
Yes good call.